src/Microsoft.ML.TimeSeries/STL/MathsTools.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.TimeSeries
+{
+    internal class MathUtility
+    {
+        /// <summary>
+        /// calculate the standard sigmoid function
+        /// </summary>
+        /// <param name="x">the input value</param>
+        public static double Sigmoid(double x)
+        {
+            return 1.0 / (1.0 + Math.Exp(-x));
+        }
```

Comment:
This method was only used by the classes you deleted in a recent commit. Can you remove this, since it's no longer used?

Reply:
removed, thanks.